product-service/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "product-service",
+  "version": "1.0.0",
+  "description": "",
+  "main": "handler.js",
+  "dependencies": {
+    "babel": "^6.23.0",
+    "babel-core": "^6.26.3",
+    "babel-preset-es2015": "^6.24.1",
+    "jest": "^26.6.3"
```

Comment:
Just FYI: those dependencies should be moved to "devDependencies"

Reply:
Thanks, according to swagger, i don't have a url, but swagger doc was generated and it is available inside an app